crates/router/src/core/payments/flows/psync_flow.rs

```diff
@@ -174,12 +196,14 @@ impl types::RouterData<api::PSync, types::PaymentsSyncData, types::PaymentsRespo
         } else {
             // in trigger, call connector for every capture_id
             for connector_capture_id in pending_connector_capture_id_list {
-                self.request.connector_transaction_id =
+                // TEMPORARY FIX: remove the clone on router data after moving this function as an impl on
```

Comment:
I think this comment is incomplete? 

Should we also consider creating an issue for this, so that we track these deferred tasks in one place?

Reply:
Done
https://github.com/juspay/hyperswitch/issues/4644